ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -96,8 +97,13 @@ data TDBTableOp
  Errors
 -------------------------------------------------------------------------------}
 
-data DbErr = DbErrStr !String
+data DbErr =
+    DbErrStr !String
   | DbErrNoSettings
+  | DbErrNoDbState
+  -- | The sequence number of a @`Db`@ should be monotonically increasing
+  -- across calls to @`bsWrite`@, since we use @`bsWrite`@ to flush
+  -- /immutable/ changes.
```

Comment:
Here we might want to explain how are the monotonically increasing property and the flushing related. I guess this has to do with the fact that we cannot flush "older" sequence numbers. Maybe we could add a small clarifying sentence.